architecture/aap-provisioning/operations.md

```
@@ -331,57 +139,39 @@ kubectl get computeinstance <name> -o json | jq -r '.status.jobs // [] | map(sel
-
-# Get all jobs by type
-kubectl get computeinstance <name> -o json | jq '.status.jobs // [] | map(select(.type == "provision"))'
 ```
 
-### Key Log Messages (AAP Direct)
-
-The AAP Direct Provider logs detailed messages about job lifecycle:
+### Key Log Messages
 
 **Provision Flow:**
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)